mysql-test/t/percona_utility_user.test

```diff
@@ -157,3 +157,33 @@ connect(plug_con,localhost,plug,frank);
 connection default;
 
 DROP USER plug;
+
+# Ensure invisibility of the utility user
```

Comment:
Please double-check that this test passes with `--repeat=N` and recorded counter values are not affected.